ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -29,44 +29,39 @@ <h1 class="ons-u-fs-3xl bulletin-header__heading">
         </div>
 
         {% block release_meta %}
-            <div class="ons-container">
-                <div class="ons-grid ons-grid-flex-gap ons-grid-flex-gap--40 bulletin-header__releases">
-                    <div class="ons-grid__col ons-col-4@m">
-                        <span class="ons-u-fs-r--b bulletin-header__releases-label">{{ _("Release date") }}:</span>
-                        <span class="ons-u-nowrap">{{ page.release_date|date("j F Y") }}</span>
-                    </div>
-                    <div class="ons-grid__col ons-col-4@m">
-                        <span class="ons-u-fs-r--b bulletin-header__releases-label">{{ _("Version") }}:</span>
-                        {% if latest_version_url %}
-                            <span>{{ _("This has been superseded.") }} <a href="{{ latest_version_url }}">{{ _("View corrected version") }}</a></span>
-                        {% else %}
-                            <span class="ons-u-nowrap">{{ _("Latest") }}</span>
-                        {% endif %}
-                    </div>
-                    <div class="ons-grid__col ons-col-4@m">
-                        {% if page.contact_details %}
-                            <span class="ons-u-fs-r--b bulletin-header__releases-label">{{ _("Contact") }}:</span>
-                            <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.name }}</a>
-                        {% endif %}
-                    </div>
+            <div class="ons-container bulletin-header__releases">
```

Comment:
The changes here are hard to read but they are basically:

1) re-order the items
2) Remove the design system flex classes and instead use a custom grid layout, which reflows top to bottom at desktop